Strategic/PreBattle Interface.cpp

```diff
@@ -482,31 +495,70 @@ void InitPreBattleInterface( GROUP *pBattleGroup, BOOLEAN fPersistantPBI )
 	fMapScreenBottomDirty = TRUE;
 	ChangeSelectedMapSector( gubPBSectorX, gubPBSectorY, gubPBSectorZ );
 	// Headrock: Added FALSE argument, We might need TRUE but not sure. Will need to initiate battle :)
-	RenderMapScreenInterfaceBottom( FALSE );
+	//RenderMapScreenInterfaceBottom( FALSE );

```

Comment:
If this function call is no longer needed, simply delete it and the accompanying comment related to it. There's plenty of old, commented out code in the repo. We don't need to add to it :)

Reply:
Cool cool, deleted.

Localization: https://github.com/1dot13/gamedir-languages/pull/10
BTW looks like Dutch PREBATTLEPANEL_DUTCH.STI was in already only in English, so I just added other resolution variants also in English for Dutch in PR.